publication-commons/src/main/java/no/unit/nva/publication/commons/customer/JavaHttpClientCustomerApiClient.java

```diff
@@ -26,6 +30,17 @@ public JavaHttpClientCustomerApiClient(final HttpClient httpClient,
         this.cognitoCredentials = cognitoCredentials;
     }
 
+    @JacocoGenerated
+    public static JavaHttpClientCustomerApiClient defaultInstance() {
```

Comment:
I think we do not need this client actually, as what it does - fetching customer. We have IdentityServiceClient in commons that should be used for this purpose. Should consider to refactor it. 
Now I am simply moving this validation from UpdatePublicationHandler to CreateUploadHandler.